lists/sz/sz-cit.json

```
@@ -0,0 +1,46 @@
+{
+  "name": {
+    "en": "Ministry of Commerce, Industry and Trade",
+    "local": ""
+  },
+  "url": "https://www.gov.sz/index.php/ministries-departments/minnistry-of-commerce",
+  "description": {
+    "en": "Companies and non-governmental organisations (NGOs) are registered with the Government of Eswatini (Swaziland)'s Ministry of Commerce, Industry and Trade. They receive a registration ID, which does not change over time.\n\nA full list of services that the Ministry provides can be found at https://www.gov.sz/index.php/services-sp-831484471, and specific information on company registration here: https://www.gov.sz/index.php/departments-sp-1596706154?id=522. "
+  },
+  "coverage": [
+    "SZ"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company"
```

```suggestion
    "company",
    "charity"
```
We use 'charity' as synonymous with NGO. Not ideal, I know.